docker/tpu/Dockerfile.incremental

```diff
@@ -1,4 +1,4 @@
-ARG IMAGE=ghcr.io/rjpower/levanter
+ARG IMAGE=ghcr.io/stanford-crfm/levanter-base
```

Comment:
I run into this when launching jobs in the shared instance. Is there any particular step that we need to run to get authorization with ghcr.io/stanford-crfm/levanter-base?

```
ERROR: failed to solve: ghcr.io/stanford-crfm/levanter-base:latest: failed to resolve source metadata for ghcr.io/stanford-crfm/levanter-base:latest: failed to authorize: failed to fetch anonymous token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Astanford-crfm%2Flevanter-base%3Apull&service=ghcr.io: 401 Unauthorized
```